src/services/car/inventory.ts

```diff
@@ -31,29 +31,44 @@ const sanitizeListing = (json): Listing => {
   }
 }
 
-export const fetchListing = async (id: number): Promise<Listing> => {
-  const listing = await fetchPath(Service.CAR, `listings/${id}`)
+export const fetchListing = async ({
+  id,
+  options = {},
+}: {
+  id: number
+  options?: RequestOptions
+}): Promise<Listing> => {
+  const listing = await fetchPath({ path: `listings/${id}`, options })
 
   return sanitizeListing(listing)
 }
 
-export const fetchDealerMakes = async (
+export const fetchDealerMakes = async ({
+  dealerId,
+  options = {},
+}: {
   dealerId: number
-): Promise<Array<{ make: string; makeKey: string }>> => {
-  return fetchPath(Service.CAR, `inventory/dealers/${dealerId}/makes`)
+  options?: RequestOptions
```

Comment:
```suggestion
  options?: ApiCallOptions
```

I just realized we never use the word `request` in this context so far. maybe align with existing naming?